examples/gfpgan/config.yaml

```diff
@@ -0,0 +1,83 @@
+data_dir: data
+environment_variables: {}
+examples_filename: examples.yaml
+input_type: Any
+model_class_filename: restoration_model.py
+model_class_name: RestorationModel
+model_framework: custom
+model_metadata: {}
+model_module_dir: model
+model_name: null
+model_type: custom
+python_version: py39
+requirements:
+- absl-py==1.1.0
+- addict==2.4.0
+- boto3==1.24.30
+- cachetools==5.2.0
+- certifi==2022.6.15
+- charset-normalizer==2.1.0
+- cycler==0.11.0
+- facexlib==0.2.4
+- filterpy==1.4.5
+- fonttools==4.34.4
+- future==0.18.2
+- google-auth==2.9.1
+- google-auth-oauthlib==0.4.6
+- grpcio==1.47.0
+- idna==3.3
+- imageio==2.19.3
+- importlib-metadata==4.12.0
+- kiwisolver==1.4.3
+- llvmlite==0.38.1
+- lmdb==1.3.0
+- Markdown==3.3.7
+- matplotlib==3.5.2
+- networkx==2.8.4
+- numba==0.55.2
+- numpy==1.20.3
+- oauthlib==3.2.0
+- opencv-python==4.6.0.66
+- packaging==21.3
+- Pillow==9.2.0
+- protobuf==3.19.4
+- pyasn1==0.4.8
+- pyasn1-modules==0.2.8
+- pyparsing==3.0.9
+- python-dateutil==2.8.2
+- PyWavelets==1.3.0
+- PyYAML==6.0
+- requests==2.28.1
+- requests-oauthlib==1.3.1
+- rsa==4.8
+- scikit-image==0.19.3
+- scipy==1.8.1
+- six==1.16.0
+- tb-nightly==2.10.0a20220713
+- tensorboard-data-server==0.6.1
+- tensorboard-plugin-wit==1.8.1
+- tifffile==2022.5.4
+- torch==1.12.0
+- torchvision==0.13.0
+- tqdm==4.64.0
+- typing_extensions==4.3.0
+- urllib3==1.26.10
+- Werkzeug==2.1.2
+- yapf==0.32.0
+- zipp==3.8.1
+- git+https://github.com/basetenlabs/BasicSR.git
+- gfpgan==1.3.4
+- realesrgan==0.2.5.0
+resources:
+  cpu: 500m
+  memory: 512m
+  use_gpu: false
+secrets:
+  gfpgan_aws_access_key_id: null
+  gfpgan_aws_secret_access_key: null
+  gfpgan_aws_region: null # e.g. us-east-1
```

Comment:
We could simply put `us-east-1` here. It just acts as default, and will be overridden by the secret.